### PR TITLE
Completed Unit model

### PR DESCRIPTION
**Pull request by KelsenH**

### CODE REVIEW

File: sojo-api/app/models/Unit.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const mongodb = require ('@onehilltech/blueprint-mongodb')
+;
+
+var schema = new mongodb.Schema({
```
Comment (technetos) [inline sojo-api/app/models/Unit.js]:
```Node
var schema = new mongodb.Schema({
```
should be
```Node
var schema = new mongodb.Schema ({
```

File: sojo-api/app/models/Unit.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const mongodb = require ('@onehilltech/blueprint-mongodb')
+;
+
+var schema = new mongodb.Schema ({
+  apartment_complex_id : {type: ObjectId, required: true, trim: true},
+  building_index       : {type: String, required: false, trim: true},
+  apartment_index      : {type: String, required: true, trim: true}
+});
```
Comment (technetos) [inline sojo-api/app/models/Unit.js]: I was wrong, when you get the chance please add back the lease_id field

**Comment by KelsenH** [comment]
Changes made

File: sojo-api/app/models/Unit.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const mongodb = require ('@onehilltech/blueprint-mongodb')
+;
```
Comment (technetos) [inline sojo-api/app/models/Unit.js]:
```Node
const mongodb   = require ('@onehilltech/blueprint-mongodb')
,     String    = mongodb.Schema.Types.String
,     Number    = mongodb.Schema.Types.Number
,     ObjectId  = mongodb.Schema.Types.ObjectId
;
```
Please update to match the above

**Pull request merged**